templates/template.html

```diff
@@ -49,6 +49,9 @@
         <li class="nav-item">
           <a class="nav-link" href="/donation">Donation</a>
                </li>
+           <li class="nav-item">
+          <a class="nav-link" href="/recent">recently_uploaded</a>
```

Comment:
@sanoussibarkay 

`recently_uploaded `is like a variable name,  use more human readable name, such as `Recent Articles`.

**Hui**